this patch will disable some handling in the core Makefile
as we use for Linux FHS a different handling in gentoo
demoplugins from core vdr install disabled
support from Linguas variable

Signed-off-by: Joerg Bornkessel <hd_brummy@gentoo.org> 2021/Feb/11
diff -Naur vdr-2.4.6.orig/Makefile vdr-2.4.6/Makefile
--- vdr-2.4.6.orig/Makefile	2020-06-27 11:13:04.000000000 +0200
+++ vdr-2.4.6/Makefile	2020-08-08 18:16:02.519719542 +0200
@@ -193,7 +193,7 @@
 
 PODIR     = po
 LOCALEDIR = locale
-I18Npo    = $(wildcard $(PODIR)/*.po)
+I18Npo    = $(foreach dir,$(LINGUAS),$(wildcard $(PODIR)/$(dir)*.po))
 I18Nmo    = $(addsuffix .mo, $(foreach file, $(I18Npo), $(basename $(file))))
 I18Nmsgs  = $(addprefix $(LOCALEDIR)/, $(addsuffix /LC_MESSAGES/vdr.mo, $(notdir $(foreach file, $(I18Npo), $(basename $(file))))))
 I18Npot   = $(PODIR)/vdr.pot
@@ -220,7 +220,7 @@
 
 install-i18n: i18n
 	@mkdir -p $(DESTDIR)$(LOCDIR)
-	cp -r $(LOCALEDIR)/* $(DESTDIR)$(LOCDIR)
+	@(cd $(LOCALEDIR); for linguas in $(LINGUAS); do [ "$$linguas" = "en" ] && continue; cp -r --parents $$linguas* $(DESTDIR)$(LOCDIR); done)
 
 # The 'include' directory (for plugins):
 
@@ -294,12 +294,11 @@
 
 install-dirs:
 	@mkdir -p $(DESTDIR)$(VIDEODIR)
-	@mkdir -p $(DESTDIR)$(CONFDIR)
 	@mkdir -p $(DESTDIR)$(ARGSDIR)
-	@mkdir -p $(DESTDIR)$(CACHEDIR)
 	@mkdir -p $(DESTDIR)$(RESDIR)
 
 install-conf: install-dirs
+	@mkdir -p $(DESTDIR)$(CONFDIR)
 	@cp -pn *.conf $(DESTDIR)$(CONFDIR)
 
 # Documentation:
@@ -307,9 +306,9 @@
 install-doc:
 	@mkdir -p $(DESTDIR)$(MANDIR)/man1
 	@mkdir -p $(DESTDIR)$(MANDIR)/man5
-	@gzip -c vdr.1 > $(DESTDIR)$(MANDIR)/man1/vdr.1.gz
-	@gzip -c vdr.5 > $(DESTDIR)$(MANDIR)/man5/vdr.5.gz
-	@gzip -c svdrpsend.1 > $(DESTDIR)$(MANDIR)/man1/svdrpsend.1.gz
+	@cp vdr.1 $(DESTDIR)$(MANDIR)/man1/
+	@cp vdr.5 $(DESTDIR)$(MANDIR)/man5/
+	@cp svdrpsend.1 $(DESTDIR)$(MANDIR)/man1/
 
 # Plugins:
 
@@ -326,8 +325,9 @@
 # Includes:
 
 install-includes: include-dir
-	@mkdir -p $(DESTDIR)$(INCDIR)
-	@cp -pLR include/vdr include/libsi $(DESTDIR)$(INCDIR)
+	@mkdir -p $(DESTDIR)$(INCDIR)/vdr $(DESTDIR)$(INCDIR)/vdr/libsi
+	@cp -pLR include/vdr $(DESTDIR)$(INCDIR)
+	@cp -pLR include/libsi Make.config $(DESTDIR)$(INCDIR)/vdr
 
 # pkg-config file:
 
